web/src/containers/activity/EntryDetails.js

```diff
@@ -28,72 +24,73 @@ const makeTitle = ({ name, fundingSource }, i) => {
   return title;
 };
 
-const components = [
-  Overview,
-  Goals,
-  Schedule,
-  Costs,
-  ContractorResources,
-  CostAllocate,
-  StandardsAndConditions
-];
+const EntryDetails = ({ activity, aKey, num }) => {
+  const [collapsed, setCollapsed] = useState(num > 1);
 
-class EntryDetails extends Component {
-  handleChange = key => () => {
-    const { toggleSection } = this.props;
-    toggleSection(key);
-  };
+  const title = useMemo(() => makeTitle(activity, num), [
+    activity.fundingSource,
+    activity.name,
+    num
+  ]);
 
-  render() {
-    const { activity, aKey, expanded, num, removeActivity } = this.props;
-    const title = makeTitle(activity, num);
+  const editContent = useMemo(
+    () => (
+      <div className="nowrap">
+        <Button
+          size="small"
+          variation="transparent"
+          onClick={() => setCollapsed(!collapsed)}
+        >
+          {collapsed ? (
+            'Edit'
+          ) : (
+            <Fragment>
+              <TimesCircle /> Close
+            </Fragment>
+          )}
+        </Button>
+      </div>
+    ),
+    [collapsed]
+  );
 
-    return (
-      <Collapsible
-        id={`activity-${aKey}`}
-        title={title}
-        btnBgColor="blue"
-        btnColor="white"
-        open={expanded}
-        onChange={this.handleChange(aKey)}
-      >
-        {components.map((Comp, i) => (
-          <Comp key={i} aKey={aKey} />
-        ))}
-        {num > 1 && (
-          <DeleteButton
-            remove={() => removeActivity(aKey)}
-            resource="activities.delete"
-          />
-        )}
-      </Collapsible>
-    );
-  }
-}
+  return (
+    <div
+      id={`activity-${aKey}`}
+      className={`activity--body activity--body__${
+        collapsed ? 'collapsed' : 'expanded'
+      } activity--body__${num === 1 ? 'first' : 'notfirst'}`}
```

Comment:
lulz

We might be able to nix this by using the `:first-child` pseudoclass, but it'd require changing the markup, and I think I want to be able to look at the whole thing in the design system before I can really suggest optimal approaches.